app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] You forgot to set up the associations between `user , comment, post` you should do that for example the comment can belong to the user and the post  many likes, comments, and posts so it should be the code like this :
```RUBY
`belongs_to :author, class_name: 'User'`
```